examples/cloud_discovery_service/library_api/c++11/README.md

```diff
@@ -68,24 +68,24 @@ in the Cloud Discovery Service User's Manual.
 
 By default, CMake will generate build files using the most common generator for
 your host platform (e.g., Makefiles on Unix-like systems and Visual Studio
-solution on Windows), \. You can use the following CMake variables to modify the
+solution on Windows). You can use the following CMake variables to modify the
```

Comment:
```suggestion
solutions on Windows). You can use the following CMake variables to modify the
```

Reply:
I agree. Since the we use the term 'Makefiles', we should be using 'Visual Studio Solutions' as a plural.